public/index.html

```diff
@@ -1,9 +1,11 @@
 <!doctype html>
 <html>
   <head>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```

Comment:
we made a change to index.html too. we added the CDN for jquery, which bootstrap also needs. maybe add it here and we can discard our change